### ISSUE TITLE
Create firewall manifest

### ISSUE THREAD

**Issue by yakirgb** [yakirgb/puppet-scylladb#1]
Add option to firewall
Something like:
```
  firewalld::custom_service{$firewalld_scylla_service_name:
      short       => $firewalld_scylla_service_name,
      port        => [
        {
          'port'     => $::scylla::params::native_transport_port,
          'protocol' => 'tcp',
        },
        {
          'port'     => $::scylla::params::rpc_port,
          'protocol' => 'tcp',
        },
        {
          'port'     => $::scylla::params::api_port,
          'protocol' => 'tcp',
        },
        {
          'port'     => $::scylla::params::jmx_port,
          'protocol' => 'tcp',
        },
      ]
    }
    -> firewalld_service{ 'Allow scylla access from the internal zone':
      ensure  => 'present',
      service => $firewalld_scylla_service_name,
      zone    => 'internal'
    }
```

**Comment by yakirgb** [comment]
#2

**Comment by yakirgb** [comment]
Closes #2
